core/lib/globals_shared.h

```diff
@@ -1246,8 +1251,9 @@ typedef char liststring_t[MAX_LIST_OPTION_LENGTH];
  * share/config.c) to set up new eventlogs (mainly for vista where our
  * installer doesn't work yet xref case 8482).*/
 #    define L_EVENT_FILE_VALUE_NAME L"File"
-#    define L_EVENT_FILE_NAME_PRE_VISTA \
-        L"%SystemRoot%\\system32\\config\\" L_EXPAND_LEVEL(EVENTLOG_NAME) L".evt"
+#    define L_EVENT_FILE_NAME_PRE_VISTA                                          \
```

Comment:
Not a review comment, but just curious what are these defines for?

Reply:
The changes are clang-format doing sthg different: not sure why.  For Windows notifications are sent to the event log (a Windows OS feature), per `-syslog_mask`, and these control aspects of that.